power_grid_model_c/power_grid_model/include/power_grid_model/calculation_parameters.hpp

```diff
@@ -126,9 +126,12 @@ struct TransformerTapRegulatorCalcParam {
     IntS status{};
 };
 
-struct TransformerTapRegulatorOptimizerOutput {
-    IntS tap_pos{na_IntS};
+struct TransformerTapPosition {
+    Idx2D index;
```

Comment:
maybe:
```suggestion
    Idx2D components_index;
```
?

Reply:
Hmm, if you mean to use `component_index` to distinguish it from a regular scalar id, perhaps? Otherwise, I am not so sure here...

Reply:
more like to distinguish explicitly index from math id, but i guess it's fine as is. let's close it